src/cf-core/usage.md

```diff
@@ -59,11 +88,11 @@ Ex. to set your base font size, add `@base-font-size-px: 17px;` to your project.
 ```
 
 
-## Media Queries
+## Media queries
 
 Mixins for consistent media queries that take `px` values and convert them to `em`s.
 
-### Min and max width media queries
+### Respond to min and max mixins
```

Comment:
My mistake, I was just trying to make all these mixin headings consistent and deleted too much here.